src/components/src/main/java/org/apache/jmeter/visualizers/SamplerResultTab.java

```diff
@@ -263,6 +263,10 @@ public void setupTabPane() {
                         .append(JMeterUtils
                                 .getResString("view_results_thread_name")) //$NON-NLS-1$
                         .append(sampleResult.getThreadName()).append(NL);
+                statsBuff
```

Comment:
This should be mentioned in the title of this PR. Was this intended to be included?

Reply:
Ah, yes, this was an later add-on. Purpose is: dealing with a long samplers listing in a View Results Tree one can easily lose relation between scroll on left side and selected sampler on right side.
Having the sampler name included can help in tracking between the 2 sources of information.
I will update PR title.